test/functional/plugin/health_spec.lua

```diff
@@ -48,68 +49,78 @@ describe('health.vim', function()
     command("set runtimepath+=test/functional/fixtures")
   end)
 
-  it("health#report_*()", function()
-    helpers.source([[
-      let g:health_report = execute([
-        \ "call health#report_start('Check Bar')",
```

Comment:
does it make sense to keep this test, but add a `health#render()` or something, that returns the formatted final result?

Reply:
I don't see the need, we are already testing all the functions render in the other functional tests that do `:chackealth sucess broken`